openshift/4.0/pipelines/Jenkinsfile.cicd

```diff
@@ -296,7 +225,23 @@ pipeline {
         echo '...'
       }
     }
+    stage('Approve Deployment') {
+      when {
+        beforeInput true
+         expression { 
+             DESTINATION == 'uat' //only ask for approval for UAT and PROD DESTINATION
```

Comment:
comment here says approval is required for UAT and PROD but only checks for uat.

Reply:
@devinleighsmith  PROD has it own seperate pipeline Jenkinsfile.cicd.prod thats why

Reply:
This pipeline us for dev, test and uat